examples/gaussian-ssm/script.jl

```diff
@@ -1,13 +1,72 @@
 # # Gaussian State Space Model
 using AdvancedPS
-using AbstractMCMC
+using Random
 using Distributions
-using Libtask: produce
-using StatsFuns
-
 using Plots
 
-x = [1, 2, 3]
-y = [2, 5, 1]
+# Parameters
+θ = @NamedTuple begin
+    a::Float64
+    q::Float64
+    r::Float64
+end
+
+a = 0.9   # Scale
+q = 0.32  # State variance
+r = 1     # Observation variance
+Tₘ = 300  # Number of observation
+Nₚ = 50   # Number of particles
+Nₛ = 1000 # Number of samples
+seed = 9  # Reproduce everything
+
+θ₀ = θ((a, q, r))
+
+f(x, t) = Normal(a * x, q) # Transition density
```

Comment:
Perhaps take parameters from `m::NonLinearTimeSeries` instead of a global variable? 

```suggestion
f(m::NonLinearTimeSeries, t) = Normal(m.θ.a * m.X[t], model.θ.q) # Transition density
```